src/openforms/js/components/admin/form_design/AuthPluginAutoLoginField.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FormattedMessage} from 'react-intl';
+
+import Field from '../forms/Field';
+import {Checkbox} from '../forms/Inputs';
+
+const AuthPluginAutoLoginField = ({availablePlugins, selectedPlugins, selectedPlugin, onChange, errors}) => {
+    let filteredPlugins = availablePlugins.filter(plugin => selectedPlugins.includes(plugin.id))
+    const authCheckboxes = filteredPlugins.map(plugin => {
+        return (
+            <li key={plugin.id}>
+                <Checkbox
```

Comment:
should be a radio, see the screenshot I posted. And to be able to reset it to no value, there should be an empty radio option added with `value=""`.